ambari-server/src/main/java/org/apache/ambari/server/utils/JsonUtils.java

```diff
@@ -27,8 +32,13 @@
  */
 public class JsonUtils {
 
+  /**
+   * Used to serialize to/from json.
+   */
   public static JsonParser jsonParser = new JsonParser();
 
+  private static final ObjectMapper JSON_SERIALIZER = new ObjectMapper();
+
```

Comment:
Probably we should use the immutable ObjectReader / ObjectWriter classes if we want to make this a general utility. ObjectMapper is said to be thread safe but uses internal locking that can be a perf bottleneck (according to StackOverflow).

Reply:
Done.